scripts/backup

```diff
@@ -1,27 +1,65 @@
 #!/bin/bash
-# vim:set noexpandtab:
 
-# Exit on command errors and treat unset variables as an error
-set -eu
+#=================================================
+# GENERIC START
+#=================================================
+# IMPORT GENERIC HELPERS
+#=================================================
 
-# Source app helpers
+source ../settings/scripts/_common.sh
```

Comment:
? source .common.sh is enough :)

Reply:
This is inspired by this:

https://github.com/YunoHost/example_ynh/blob/master/scripts/backup#L9

Backup and restore scripts are not like other scripts and do not run in the same place.

Reply:
oki :) It's just me (I'm so tired :/). Well, I didn't even check what was failing. Do you want to take care of this app with me ?

Reply:
Yeah if you want!

Reply:
Maybe I can upgrade the Readme script to be like this one?
https://github.com/YunoHost/example_ynh/blob/master/README.md

Reply:
ok. I added you as admin of this repo. Better to be two than alone (I see so much app which are abandoned by their maintainer and taken again 1 year later).